configure.ac

```diff
@@ -274,21 +274,61 @@ AC_TYPE_SIZE_T
 AC_TYPE_UINT64_T
 AC_LANG_POP(C++)
 
+AS_IF([test "$want_pi" = yes], [
```

Comment:
If we are going to end up with all of this here anyway, perhaps my earlier suggestion of using `AC_CONFIG_SUBDIRS` and keeping `targets/simple_switch_grpc/configure.ac` as is doesn't make sense any more?

Reply:
Using `AC_CONFIG_SUBDIRS` makes sense in the context of the previous pull request (https://github.com/p4lang/behavioral-model/pull/1105). It reduces the amount of changes necessary to integrate simple_switch_grpc with `configure.ac` at the top-level directory, and the risk of breaking something or introducing unexpected bugs. Once we add P4Runtime support in psa_switch, we can reuse most of the code from `targets/simple_switch_grpc/configure.ac` for both targets. IMHO reusing common code instead of duplicating it would make it more maintainable and easier to test in the future. If you think this is a good idea, I can add this change in the pull request adding P4Runtime support to psa_switch.

Reply:
> If you think this is a good idea, I can add this change in the pull request adding P4Runtime support to psa_switch.

That sounds good to me